src/components/ECharts/ChainChart/index.tsx

```diff
@@ -552,11 +673,11 @@ export default function AreaChart({
 				{
 					...yAxis,
 					axisLabel: {
-						formatter: (value) => toK(value) + ' ' + 'Addr',
+						formatter: (value) => toK(value) + ' ' + 'Users',
```

Comment:
using addresses rather than users was a conscious choice since theres a bunch of users using multiple addresses